clients/shared/utils_test.go

```diff
@@ -0,0 +1,108 @@
+package shared
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+
+	bigQueryDialect "github.com/artie-labs/transfer/clients/bigquery/dialect"
+	redshiftDialect "github.com/artie-labs/transfer/clients/redshift/dialect"
+	snowflakeDialect "github.com/artie-labs/transfer/clients/snowflake/dialect"
+	"github.com/artie-labs/transfer/lib/sql"
+	"github.com/artie-labs/transfer/lib/typing"
+	"github.com/artie-labs/transfer/lib/typing/columns"
+	"github.com/artie-labs/transfer/lib/typing/ext"
+)
+
+var dialects = []sql.Dialect{
+	bigQueryDialect.BigQueryDialect{},
+	redshiftDialect.RedshiftDialect{},
+	snowflakeDialect.SnowflakeDialect{},
+}
+
+func TestColumn_DefaultValue(t *testing.T) {
```

Comment:
This is pretty much what we had before when we had a single test for `DefaultValue`.